sql/src/main/java/org/apache/druid/sql/calcite/planner/convertlet/TimeInIntervalConvertletFactory.java

```diff
@@ -55,7 +55,7 @@ public class TimeInIntervalConvertletFactory implements DruidConvertletFactory
       .operatorBuilder(NAME)
       .operandTypeChecker(
           OperandTypes.sequence(
-              "'" + NAME + "(<TIMESTAMP>, <LITERAL ISO8601 INTERVAL>)'",
+              NAME + "(<TIMESTAMP>, <LITERAL ISO8601 INTERVAL>)",
```

Comment:
it seems like a handful of functions define their signature with single quotes around it, why are we removing them here only?